source/Engine.cpp

```diff
@@ -243,6 +247,16 @@ namespace {
 
 		return make_pair(newCenter, newVelocity);
 	}
+
+	const Color &GetGunsightColor(const Ship &ship)
+	{
+		if(ship.IsYours())
+			return *GameData::Colors().Get("gunsight friendly");
+		else if(ship.GetTargetShip() && (ship.GetTargetShip()->IsYours() || ship.GetGovernment()->IsEnemy()))
```

Comment:
Shouldn't be both (in case the ship is targeting you to assist)?
```suggestion
		else if(ship.GetTargetShip() && ship.GetTargetShip()->IsYours() && ship.GetGovernment()->IsEnemy())
```